scripts/analyzer/main.py

```diff
@@ -29,17 +30,18 @@ def check_installation(package) -> bool:
             check=True,
         )
         # I learned to use a PIPE with the subprocess module here: https://stackoverflow.com/questions/13332268/how-to-use-subprocess-command-with-pipes
-    except subprocess.CalledProcessError:
+    except subprocess.CalledProcessError or FileNotFoundError:
         #  The package isn't installed, then return false: https://stackoverflow.com/questions/32942207/python-subprocess-calledprocesserror-command-returned-non-zero-exit-s
         return False
     return True
 
 
-def execute_chasten(search_path, save_directory, save_file_path, chasten_config_path):
+def execute_chasten(search_path, save_directory, chasten_config_path):
```

Comment:
are we adding `execute_chasten` twice here?

Reply:
fixed